setup.py

```diff
@@ -25,7 +25,7 @@ def read_file(name):
 
 DEPS = [
     'ludolph-zabbix>=1.5',
-    'sleekxmpp>=1.2.0',
+    'sleekxmpp==1.3.2',
```

Comment:
I'm not sure whether this is a good idea. After a new release of SleekXMPP we would have to release a new version of Ludolph. This will also lock the user's environment to this version.  I suggest to keep the minimal version to `1.2.0` and maybe add `1.4.0` as the upper limit.